src/download/src/lib.rs

```diff
@@ -277,8 +277,10 @@ pub mod reqwest_be {
         let buffer_size = 0x10000;
         let mut buffer = vec![0u8; buffer_size];
 
-        if let Some(len) = res.headers().get::<header::ContentLength>() {
-            callback(Event::DownloadContentLengthReceived(len.0 + resume_from))?;
+        if let Some(len) = res.headers().get(header::CONTENT_LENGTH) {
+            // TODO possible issues during unwrap?
+            let len = len.to_str().unwrap().parse::<u64>().unwrap() + resume_from;
```

Comment:
What is the purpose of converting the header to a string and back?

Reply:
`len` is `&reqwest::header::HeaderValue` (not `u64`) there IIRC so I need to convert it to `str` and then parse it as `u64`.

Even though `HeaderValue` have `HttpTryFrom` but `TryFrom` is nigthly so not usable.

I removing the extra `to_str` but `HeaderValue` does not implement `parse`.

Maybe I might have missed something?